app/assets/javascripts/wpcc/components/SalaryConditions.js

```diff
@@ -38,6 +20,7 @@ define(['jquery', 'DoughBaseComponent'], function($, DoughBaseComponent) {
     this.contribution = 'full';
 
     // Step 2 - Contributions
+    this.optInTriggers = config;
```

Comment:
It would be good to document somewhere how dough uses html data-attributes.

Reply:
Done on the PR and in the commit message